sklearn/ensemble/tests/test_iforest.py

```diff
@@ -263,14 +263,17 @@ def test_iforest_subsampled_features():
 def test_iforest_average_path_length():
     # It tests non-regression for #8549 which used the wrong formula
     # for average path length, strictly for the integer case
+    # Updated to test non-regression for #11839.
 
     result_one = 2. * (np.log(4.) + euler_gamma) - 2. * 4. / 5.
     result_two = 2. * (np.log(998.) + euler_gamma) - 2. * 998. / 999.
-    assert_almost_equal(_average_path_length(1), 1., decimal=10)
+    assert_almost_equal(_average_path_length(0), 0., decimal=10)
```

Comment:
Can't we use `assert _average_path_length(0) == 0` instead of `assert_almost_equal`? Same for  `assert _average_path_length(1)` and  `assert _average_path_length(2)`.

Reply:
I guess using the sklearn test style, I should then be using "assert_equal"? It's imported in the file, so I'm going to assume that's considered to be the right way to test this.